gluetool/utils.py

```diff
@@ -725,7 +721,7 @@ def dump_yaml(data, filepath, logger=None):
 
 def _json_byteify(data, ignore_dicts=False):
     # if this is a unicode string, return its string representation
-    if isinstance(data, unicode):
+    if isinstance(data, str):
```

Comment:
`six.string_types` instead of `str`

Reply:
thanks, fixed all checks to be python2 and python3 compatible